packages/cli-tool/src/commands/generate/version-control.ts

```diff
@@ -0,0 +1,24 @@
+import * as fs from 'node:fs';
+
+import { cli } from 'cli-ux';
+
+export const VERSION_CONTROL_OPTIONS: { [key: string]: string } = {
+  github: 'GitHub',
+  gitlab: 'GitLab',
+  none: 'None',
+};
+
+export const SetVersionControl = (appName: string, versionControl: string): void => {
```

Comment:
```suggestion
export const setVersionControl = (appName: string, versionControl: string): void => {
```
Should this be lower case? (classes only are PascalCase according to [our conventions ](https://nimblehq.co/compass/development/code-conventions/javascript/#naming) — and this one is not a React component :) )